app/redux/TransactionSaga.js

```diff
@@ -154,7 +154,8 @@ function* broadcastOperation({payload:
             }
         }
         yield call(broadcast, {payload})
-        const eventType = type.replace(/^([a-z])/, g => g.toUpperCase()).replace(/_([a-z])/g, g => g[1].toUpperCase());
+        let eventType = type.replace(/^([a-z])/, g => g.toUpperCase()).replace(/_([a-z])/g, g => g[1].toUpperCase());
+        if (eventType === 'Comment' && !operation.parent_permlink) eventType = 'Post';
```

Comment:
IIRC, top level posts do have a parent_permlink (it becomes 'category'). Should this be checking for blank `parent_author` instead?